doc/GETSTARTED.md

```diff
@@ -1,88 +1,102 @@
-  cmake --build build -- -j5
-  ```
+```sh
+cd libelektra  #navigate to libelektra
+mkdir build  && cd build  #create and navigate to the build directory
+cmake ..  # watch output to see if everything needed is included
+#  optionally run "ccmake .." to get an overview of the available build settings (needs cmake-curses-gui)
+cmake --build build -- -j5
+```
 
-  Optionally you can also run tests, see [here for more information](/doc/TESTING.md):
+Optionally you can also run tests, see [here for more information](/doc/TESTING.md):
 
-  ```sh
-  cmake --build build --target run_nokdbtests
-  ```
+```sh
+cmake --build build --target run_nokdbtests
+```
 
-  With these commands you will be able to run the "Hello World!" example, but usually you will need to use some of the [plugins](/src/plugins/README.md), tools and bindings of Elektra. Please take a look at the more detailed [compiling documentation](/doc/COMPILE.md). After you completed building Elektra on your own, you can execute these commands to install Elektra (please check the [installation documentation](/doc/INSTALL.md) for the many available packages):
+With these commands you will be able to run the "Hello World!" example, but usually you will need to use some of the [plugins](/src/plugins/README.md), tools and bindings of Elektra. Please take a look at the more detailed [compiling documentation](/doc/COMPILE.md). After you completed building Elektra on your own, you can execute these commands to install Elektra (please check the [installation documentation](/doc/INSTALL.md) for the many available packages):
 
-  ```sh
-  sudo make install
-  sudo ldconfig #optional: check installation documentation for more information
-  ```
+```sh
+sudo make install
+sudo ldconfig #optional: check installation documentation for more information
+```
 
-  [Installation documentation](/doc/INSTALL.md) contains further information about available packages.
+[Installation documentation](/doc/INSTALL.md) contains further information about available packages.
 
-  Optionally you can also run tests to verify the installed Elektra, see [here for more information](/doc/TESTING.md):
+Optionally you can also run tests to verify the installed Elektra, see [here for more information](/doc/TESTING.md):
 
-  ```sh
-  kdb run_nokdbtests
-  ```
+```sh
+kdb run_nokdbtests
+```
 
-- Hello World!
+## Docker
 
-  Start with your very first Elektra application in C and follow these steps:
-  [Hello World!](/doc/tutorials/hello-elektra.md)
+In order to ease the above process, we also provide [up-to-date docker images](https://hub.docker.com/u/elektra) which you can use to try docker without having to install and run docker on your local machine.
+
+More detailed tutorials on running specific taks or tools:
+
+- [General Docker readme](/scripts/docker/README.md)
+- [Testing](/doc/tutorials/run_all_tests_with_docker.md)
+- [WebUI](/doc/tutorials/install-webui.md)
```

Comment:
Without knowing what the WebUI this link is pointless, you can also find the tutorial in the list of tutorials. I don't think it should be listed here.

Reply:
I'm not sure. Is the GETSTARTED.md a help for developers or just for any user who wants to see elektra in action. If former I agree with you, if latter I think its nice if you want to approach elektra from the UI side instead of the command line side.

Reply:
> Is the GETSTARTED.md a help for developers or just for users who want to see elektra in action.

GETSTARTED.md is meant for developers. Users usually not not need to compile Elektra. Maintainers would look at doc/COMPILE.md. This should be clarified in the beginning of each document.